src/main/java/com/walmartlabs/x12/standard/X12Group.java

```diff
@@ -27,12 +34,28 @@
     // GS05
     private String time;
     // GS06
-    private String groupControlNumber;
+    private String headerGroupControlNumber;
     // GS07
     private String responsibleAgencyCode;
     // GS08
     private String version;
 
+    // ST...SE loops within the group
+    List<X12TransactionSet> transactions;
+
+    // Trailer Data Elements
+    // GE01
+    private Integer numberOfTransactions;
+    // GE02
+    private String trailerGroupControlNumber;
+
+    public void addTransactionSet(X12TransactionSet tx) {
```

Comment:
Should we add another method public `void addTransactionSet(List<X12TransactionSet> txs)` which would add transactions to existing transaction list?

Reply:
YAGNI at the moment